src/components/Progress.js

```diff
@@ -0,0 +1,24 @@
+import { PropTypes } from 'prop-types';
+import { React } from 'react';
+
+function Progress(props) {
+  const { progress } = props;
+  return (
+    <div className="progress-container">
+      <div className="progress" style={{ backgroundImage: `conic-gradient(#0290ff ${progress}%, whitesmoke ${progress}%)` }}>
```

Comment:
- [ ] Inline styling is one of CSS bad practices. When you add inline styles, it makes it difficult revising the design of your site and makes your code harder to manage.
I would recommend moving the style out of the `div`. Because you are using a variable in the style, try to define a constant `style` in the `Progress` component and give it to the style attribute.

Reply:
Unfortunately in this case, Since I am using a dynamic style based on calculations, I can implement any other approach.

Reply:
- [ ] I understand @Behnam1369. As I said the style can be extracted out of the `div` tag as shown below.: 
```js
 const { progress } = props;
 const style = {
      backgroundImage: `conic-gradient(#0290ff ${progress}%, whitesmoke ${progress}%)` 
 }
  return (
    <div className="progress-container">
      <div className="progress" style={style}>
```

Reply:
Ok. I got it. Thanks for your explanation.